src/Components/User/Header/Header.tsx

```diff
@@ -78,24 +78,18 @@ export const Header = (headerProps: HeaderProps) => {
                   >
                     Our Beliefs
                   </NavDropdown.Item>
+                </NavDropdown>
+              </Nav.Item>
+              <Nav.Item>
+                <NavDropdown className={"header-navbar-link"} title="Resources">
                   <NavDropdown.Item
-                    activeKey={headerProps.activeKey}
                     className={"px-2 header-navbar-link"}
```

Comment:
small styling note:
can we move px-2 to be on the dropdown title Resources?
can you also do the same for the About dropdown? (remove px-2 from line 70 and 76 and move to line 68)